src/modules/dashboard/components/ColonyHome/SidebarClickableHeading/SidebarClickableHeading.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactChild } from 'react';
```

Comment:
I am not totally sure but I think that it would be better to move it to the core components...